build_config/Darwin.nag.default/ESMF_Conf.inc

```diff
@@ -34,8 +34,6 @@ If a bug fix for this is released, the next #define can
 be removed.
 #endif
```

Comment:
Comment should be removed now.

Reply:
Ah - I did not even notice that comment.   Deleted in my branch now.   Will run full tests and get back to you.  Probably tomorrow ...